Formula/imagemagick.rb

```diff
@@ -72,49 +73,51 @@ def install
       --enable-static
     ]
 
-    if build.without? "modules"
-      args << "--without-modules"
-    else
-      args << "--with-modules"
-    end
-
-    if build.with? "opencl"
-      args << "--enable-opencl"
-    else
-      args << "--disable-opencl"
-    end
-
-    if build.with? "openmp"
-      args << "--enable-openmp"
-    else
-      args << "--disable-openmp"
-    end
-
-    if build.with? "webp"
-      args << "--with-webp=yes"
-    else
-      args << "--without-webp"
+    %W[
+      hdri hdri
```

Comment:
since these are also the same, you don't have to repeat them in pairs – just use each instead of each_slice(2) and go horizontal